addons/mail/static/src/discuss/discuss_patch.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.Discuss" t-inherit-mode="extension">
```

Comment:
Not to do in this PR, just a comment to share.

We could rename this component `DiscussApp` at one point, so it's clear what it means (`discuss` being the module itself, `messaging` will be renamed to `discuss`, and the entry `discuss` in store will renamed to `discuss_app` (or heck, even removed and state is flattened).